server/graphql/v2/enum/ContributionFrequency.ts

```diff
@@ -8,6 +8,7 @@ export const GraphQLContributionFrequency = new GraphQLEnumType({
     MONTHLY: {},
     YEARLY: {},
     ONETIME: {},
+    RECURRING: {},
```

Comment:
This enum is used in other queries and mutation inputs. For this I think we should filter with another value in the OrderCollectionQuery.

Reply:
Makes sense. I've removed this and used the alternative of making `onlySubscriptions` also return processing subscriptions. Thanks. 👍🏽